examples/gpu/eemumu_AV/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -590,130 +594,120 @@ int main(int argc, char **argv)
               << "StdDevMatrixElemValue     = " << stdelem << " GeV^" << meGeVexponent << std::endl
               << "MinMatrixElemValue        = " << minelem << " GeV^" << meGeVexponent << std::endl
               << "MaxMatrixElemValue        = " << maxelem << " GeV^" << meGeVexponent << std::endl;
+  }
 
-    const std::string jsonKey = "9e DumpJson";
-    timermap.start(jsonKey);
-
-    if(json) {
-      std::ofstream jsonFile;
-      std::string perffile = std::to_string(jsondate) + "-perf-test-run" + std::to_string(jsonrun) + ".json";
-      perffile = "./perf/data/" + perffile;
-
-      //Checks if file exists
-      std::ifstream fileCheck;
-      bool fileExists = false;
-      fileCheck.open(perffile);
-      if(fileCheck){
-        fileExists = true;
-        fileCheck.close();
-      }
+  // --- 9e Dump to json
+  const std::string jsonKey = "9e DumpJson";
+  timermap.start(jsonKey);
+
+  if(json)
+  {
+    std::ofstream jsonFile;
+    std::string perffile = std::to_string(jsondate) + "-perf-test-run" + std::to_string(jsonrun) + ".json";
+    perffile = "./perf/data/" + perffile;
+
+    //Checks if file exists
+    std::ifstream fileCheck;
+    bool fileExists = false;
+    fileCheck.open(perffile);
+    if(fileCheck){
+      fileExists = true;
+      fileCheck.close();
+    }
 
-      jsonFile.open(perffile, std::ios_base::app);
+    jsonFile.open(perffile, std::ios_base::app);
 
-      if(!fileExists){
-        jsonFile << "[" << std::endl;
-      }
-      else{
-        //deleting the last bracket and outputting a ", "
-        std::string temp = "truncate -s-1 " + perffile;
-        const char *command = temp.c_str();
-        system(command);
-        jsonFile << ", " << std::endl;
-      }
+    if(!fileExists){
+      jsonFile << "[" << std::endl;
+    }
+    else{
+      //deleting the last bracket and outputting a ", "
+      std::string temp = "truncate -s-1 " + perffile;
+      const char *command = temp.c_str();
+      system(command);
+      jsonFile << ", " << std::endl;
+    }
 
-      jsonFile << "{" << std::endl
```

Comment:
Looks like just the indention changed.

Reply:
Yes this is one of the main changes, now the "if(json)" is at the same level as, and outsode of, "if(perf)", while it was inside before. The logic of the json flag was not enforced very clearly (it needed -p too to do something useful). Now I redefined -p to print to screen and -j to dump to json, but the stats are computed all the time (previosuly they were computed only within "if(perf)").